.travis.yml

```diff
@@ -42,6 +42,7 @@ jobs:
         # this or we should error if it dirties the repo.
         - ./bin/protoc-go.sh
         - go test -v ./...
+        - go vet -v ./...
```

Comment:
Tiny nit -- I find the output of this command to be more readable if you omit the `-v` flag.